### PR TITLE
add add to cart function in for-sale.html

### PR DESCRIPTION
**Pull request by MirkaaStefanov**

### CODE REVIEW

**Comment by MirkaaStefanov** [comment]
Added button for addToCart in for-sale.html underneath the make order button

**Comment by MirkaaStefanov** [comment]
Add viewing the cart can remove item, but can not change the quantity for now

File: src/main/java/com/example/LukeriaFrontendApplication/config/ShoppingCartClient.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+@FeignClient(name = "lukeria-erp-shopping-cart", url = "${backend.base-url}/shoppingCart")
+public interface ShoppingCartClient {
+
+    @PostMapping("/addToCart")
+    void addToCart(@RequestParam("productId") Long productId, @RequestParam("quantity") int quantity, @RequestHeader("Authorization") String auth);
+
+    @GetMapping("/showCart")
+    List<CartItemDTO> showCart(@RequestHeader("Authorization") String auth);
+
+    @PostMapping("/removeCartItem")
```
Comment (GeorgiStoyanovAtanasov) [inline src/main/java/com/example/LukeriaFrontendApplication/config/ShoppingCartClient.java:24]: Would probably be better to use @DeleteMapping but this also works

**Review by GeorgiStoyanovAtanasov — APPROVED** [review]

**Comment by berki0** [comment]
I left a comment about an issue in Trello

**Pull request closed without merging**